prodigy_demo/llm_recipe/README.md

```diff
@@ -0,0 +1,90 @@
+## 👽 LLM recipe
+
+Now let's explore the use of LLMs in Prodigy.
+
+Similarly, Prodigy offers built-in recipes with LLM integrations with `spacy-llm`. They have integrations with a number of LLM providers, including:
+
+- OpenAI;
+- Cohere;
+- Anthropic etc. 
+
+You can [read more about them here.](https://prodi.gy/docs/recipes#textcat-llm.correct) 
+
+However, if you would like to make minor admendments to the recipes, you can similarly customise them, making use of libraries like `langchain`.
+
+Let's explore a simple custom LLM recipe. 
+
+### 👽🛫 Classifying airline tweets: LLM edition 
+
+Let's revisit the simple **text classification task** where we would like to label tweets about airlines as `positive` or `negative` for sentiment.
+
+Instead of using a huggingface model, let's use a LLM to classify the tweets.
+
+In your prodigy environment, install additional libraries:
+
+```
+pip install -r llm_requirements.txt
+```
+Then, ensure you have the openai API key in your environment variables. 
+
+```
+echo OPENAI_KEY=<YOUR_API_KEY> >> .env
```

Comment:
```suggestion
echo OPENAI_KEY="YOUR_API_KEY" >> .env
```

Reply:
I think you need to use "" instead of <> but perhaps both work?

Reply:
good catch!